docker-images/notebook/prepare.sh

```diff
@@ -5,6 +5,19 @@ set -x
 echo "Copy files from pre-load directory into home"
 cp --update -r -v /pre-home/. /home/jovyan
 
+if [ -z "$EXAMPLES_GIT_URL" ]; then
+    export EXAMPLES_GIT_URL=https://github.com/pangeo-data/pangeo-example-notebooks
+fi
+git clone $EXAMPLES_GIT_URL examples
```

Comment:
we should only run this line if examples does not exist:

```bash
if [ ! -d "examples" ]; then
  git clone $EXAMPLES_GIT_URL examples
fi
```

Reply:
Does this not break the point of the PR?  Don't we always want to create a new examples directory?

Reply:
No - if examples exists and is a git repo, we just update it. It examples exists and is not a repo (old work), nothing happens.